cmd/autoscaler/main.go

```diff
@@ -159,19 +159,6 @@ func main() {
 		return ctl.Run(controllerThreads, stopCh)
 	})
 
-	// Setup the metrics to flow to Prometheus.
```

Comment:
Is this started implicitly by importing "metrics" above?

Reply:
Yes, this will handled by metrics.UpdateExporterFromConfigMap. If it's prometheus, this will be called.
https://github.com/knative/pkg/blob/594df37bfda1543f724e76beeb0a74f530c1de57/metrics/exporter.go#L85